Mixpanel/Mixpanel.m

```diff
@@ -1667,16 +1667,17 @@ - (void)notificationController:(MPNotificationViewController *)controller wasDis
     };
 
     if (status && controller.notification.callToActionURL) {
-        MixpanelDebug(@"%@ opening URL %@", self, controller.notification.callToActionURL);
-        BOOL success = [[UIApplication sharedApplication] openURL:controller.notification.callToActionURL];
+        [controller hideWithAnimation:NO completion:^{
```

Comment:
My only concern with this is that if a developer adds a custom URL to their notification they may not be aware that we are using this method internally and if they haven't registered the custom URL in the Info.plist then this code change will cause it to silently fail.

So the question now, is should we change this behavior using the new method and document what's happening so developers will understand, or should we use the previous `openURL:` call which seems to have the same behavior on iOS 9.